nixup/modules/nixup/nixup-resources.sh

```diff
@@ -0,0 +1,124 @@
+#! @shell@
+
+rmEmptyDirectories () {
+	BASE=$1
+	DIR=$2
+	if [ "$DIR" != "." -a -d "$BASE/$DIR" ]; then
+		if [ ! "$(@coreutils@/bin/ls -A $BASE/$DIR)" ]; then
+			@coreutils@/bin/rm -rf $BASE/$DIR
```

Comment:
Also, isn't this a possible race condition, if a file is created between the ls call and the rm call?